### PR TITLE
089 website styling

### PR DESCRIPTION
**Pull request by irinakulinich3712**
HTML structure has been slightly changes, as well as some classes have been added to easily operate the elements in CSS. Styling for the pages has been added to the CSS file.

### CODE REVIEW

**Review by adzai — COMMENTED** [review]
Looks nice, good job! 

I have only 1 problem with it - the background image is
very low resolution, could it be changed?

**Comment by irinakulinich3712** [comment]
> Looks nice, good job!
> 
> I have only 1 problem with it - the background image is
> very low resolution, could it be changed?

Sure, I'll change it now

**Review by AniSanikidze — COMMENTED** [review]
Looks really nice! Just one thing, the page for riegerovy-sady has a problem.  It's not displaying the content.

**Comment by adzai** [comment]
> Looks really nice! Just one thing, the page for riegerovy-sady has a problem. It's not displaying the content.

It's a spelling mistake. Should be riegrovy-sady

**Comment by irinakulinich3712** [comment]
> Looks really nice! Just one thing, the page for riegerovy-sady has a problem. It's not displaying the content.

Right, I've missed that one, will correct the spelling now

**Review by adzai — COMMENTED** [review]
Backgroung looks good.
Other stuff:

- The chatbot window starts out kinda small and tall

chatbot window

- Will you add the greeting and help?

**Comment by irinakulinich3712** [comment]
> Backgroung looks good.
> Other stuff:
> 
> * The chatbot window starts out kinda small and tall
> 
> chatbot window
> 
> * Will you add the greeting and help?

1. I'm fixing it now
2.  It depends on how the greeting and help elements will be implemented in web version, as I am not sure about them

**Comment by adzai** [comment]
> 2.  It depends on how the greeting and help elements will be implemented in web version, as I am not sure about them

- Greeting should be static information on the home page explaining the chatbot
in the same way as in the REPL.
- I'll probably make a separate page for help, so next to the home button there could
be a help button. I'll make a new branch for it.

**Comment by irinakulinich3712** [comment]
> > 1. It depends on how the greeting and help elements will be implemented in web version, as I am not sure about them
> 
> * Greeting should be static information on the home page explaining the chatbot
>   in the same way as in the REPL.

But I think the text of the greeting should be different, shouldn't it? As the web version functionality differs.  For example this could be the text of the greeting: "Hi! I am your park guide. You can choose the park you are interested in and write what you want to know in the chat." 

> * I'll probably make a separate page for help, so next to the home button there could
>   be a help button. I'll make a new branch for it.

Alright

**Comment by adzai** [comment]
Here #104.

File: src/web/pages.clj [hunk 1]
```diff
@@ -45,7 +45,7 @@
         [:a {:href "/bertramka" :class "btn btn-park"}
          "Bertramka"]]
        [:p {:class "btn-wrapper"}
-        [:a {:href "/riegerovy-sady" :class "btn btn-park"}
+        [:a {:href "/riegrovy-sady" :class "btn btn-park"}
          "Riegerovy Sady"]]
```
Comment (AniSanikidze) [inline src/web/pages.clj:49]: Spelling still incorrect

Comment (irinakulinich3712) [reply src/web/pages.clj:49]: riegrovy-sady is the way the name of the park is written in our data

Comment (adzai) [reply src/web/pages.clj:49]: I think it's the one under it on line 49.

Comment (irinakulinich3712) [reply src/web/pages.clj:49]: > I think it's the one under it on line 49.

What do you mean? Also, the chat window should be fine now, check it out please

**Comment by adzai** [comment]
Chat window works fine now

File: src/web/pages.clj (hunk 1 above)
Comment (AniSanikidze) [reply src/web/pages.clj]: Yes, I meant line 49. The spelling there is incorrect. On line 48 you already fixed it.

Comment (irinakulinich3712) [reply src/web/pages.clj]: > Yes, I meant line 49. The spelling there is incorrect. On line 48 you already fixed it.

Oh, got it. Will fix now

**Comment by irinakulinich3712** [comment]
> Chat window works fine now

Good. I have rebased the branch and now am working on help and history pages

**Review by adzai — APPROVED** [review]
Looks good

**Review by AniSanikidze — APPROVED** [review]

**Pull request merged**